habitat/core/gym_env_episode_count_wrapper.py

```diff
@@ -0,0 +1,50 @@
+from typing import Tuple, Union
+
+from gym import Env, Wrapper, spaces
+from gym.core import ActType, ObsType
+
+from habitat.core.dataset import Episode
+
+
+class EnvCountEpisodeWrapper(Wrapper):
```

Comment:
I wonder if this wrapper and `EnvObsDictWrapper` should be compacted into one wrapper that just converts regular Gym environments to be compatible with our trainer. Is there ever a case to use one but not the other?

Reply:
I was thinking doing it this way, but I think it makes more sense to separate the functionality into different wrappers because they do very different things. I would even encourage future functionalities to be added in new wrappers rather than editing existing wrappers. This allows for better separation of concerns and makes it harder to break existing things. I agree that it is strange since in our use cases, these wrappers will most likely always be used together.

Reply:
Sure, this makes sense to me.